topicgenerator/src/main/java/io/dockstore/topicgenerator/client/cli/TopicGeneratorClient.java

```diff
@@ -209,12 +207,23 @@ private void getAiGeneratedTopicAndRecordToCsv(OpenAiService openAiService, CSVP
         final String finishReason = chatCompletionChoice.getFinishReason();
         final long promptTokens = chatCompletionResult.getUsage().getPromptTokens();
         final long completionTokens = chatCompletionResult.getUsage().getCompletionTokens();
+        String descriptorFileChecksum = descriptorFile.getChecksum().isEmpty() ? "" : descriptorFile.getChecksum().get(0).getChecksum();
```

Comment:
Another option would be to hash the entire `ChatMessage`, especially if it's easy to dump it in string format.  If we did that, we'd be able to detect when the prompt changed, too.